src/Stratis.Bitcoin.Features.Consensus/ConsensusFeature.cs

```diff
@@ -254,15 +254,22 @@ public IEnumerable<ConsensusRule> GetRules()
             {
                 return new List<ConsensusRule>
                 {
-                    new BlockHeaderRule(),
+                    // == Header ==
```

Comment:
@dangershony these would also be needed for Smart Contract rule registration 👍

Reply:
ah yes ofcourse